pkg/render/compliance.go

```diff
@@ -1144,7 +1144,6 @@ func (c *complianceComponent) complianceBenchmarkerSecurityContextConstraints()
 		Users: []string{
 			fmt.Sprintf("system:serviceaccount:%s:tigera-compliance-benchmarker", ComplianceNamespace),
 		},
-		Groups:  []string{"system:authenticated"},
```

Comment:
I don't understand why these "Groups" fields were removed, but I assume it's needed!

Reply:
This came in from v1.21.2, the original PR that added this change https://github.com/tigera/operator/pull/1488
I've asked if this could affect and MCM setup and was told it should not.